app/app.go

```diff
@@ -657,7 +659,9 @@ func initAppModules(keepers *KeeperCache, bApp *bam.BaseApp, encodingConfig axel
 			bApp.Router(),
 		),
 		permission.NewAppModule(*getKeeper[permissionKeeper.Keeper](keepers)),
+		batch.NewAppModule(encodingConfig.Codec, bApp.MsgServiceRouter()),
```

Comment:
I don't have a very strong opinion here, but I'd prefer naming the module more genericly so that more functionality related to convinience can be added later.

Reply:
open to name suggestion